qemu/tests/cfg/cpu_model_negative.cfg

```diff
@@ -0,0 +1,24 @@
+- cpu_model_negative:
+    type = cpu_model_negative
+    start_vm = no
+    warning_msg = ''
+    variants:
+        - enforce:
+            cpu_model_flags = '+avx512_bf16,check,enforce'
+            warning_msg = "Property '.avx512_bf16' not found"
+        - smp_lt_maxcpu:
+            enforce_smp_topology = yes
+            smp = 8
+            vcpu_cores = 2
+            vcpu_threads = 2
+            vcpu_sockets = 2
+            vcpu_maxcpus = 4
+            warning_msg = 'maxcpus must be equal to or greater than smp'
+        - smp_out_of_range:
+            enforce_smp_topology = yes
+            smp = OUT_OF_RANGE
+            vcpu_cores = 0
+            vcpu_threads = 0
+            vcpu_sockets = 0
+            vcpu_maxcpus = 0
+            warning_msg = 'Number of SMP cpus requested (${smp}) exceeds max cpus supported by KVM'
```

Comment:
@vivianQizhu Hi, This is the qemu warning  info: 
[qemu output] qemu-kvm: Invalid SMP CPUs 513. The max CPUs supported by machine 'pc-q35-rhel8.3.0' is 512.

Reply:
@nanliu-r Updated, thanks.
 (1/3) Host_RHEL.m8.u3.product_rhel.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.cpu_model_negative.enforce.q35: PASS (4.08 s)
 (2/3) Host_RHEL.m8.u3.product_rhel.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.cpu_model_negative.smp_lt_maxcpu.q35: PASS (4.14 s)
 (3/3) Host_RHEL.m8.u3.product_rhel.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.cpu_model_negative.smp_out_of_range.q35: PASS (4.16 s)